src/crm/dynamicsWebApi.ts

```diff
@@ -59,26 +61,63 @@ const onRejected = (error: AxiosError) => {
   return Promise.reject(error);
 };
 
-const getTestStationEntities = async (requestUrl: string): Promise<DynamoTestStation[]> => {
+const getAccountsEntities = async (requestUrl: string): Promise<DynamicsTestStation[]> => {
```

Comment:
Wouldn't a call to `getAccountEntities` return a type that had `account` in it's name?
```suggestion
const getAccountEntities = async (requestUrl: string): Promise<DynamicsAccount[]> => {
```

Reply:
I think it's because I had DynamoTestStation and DynamicsTestStation and wanted to illustrate that you're mapping one format (dynamics) to another (dynamo)